docs/millennium-os/manual/chapters/30_usage.md

```diff
@@ -1,32 +1,34 @@
+That said, there are several different probing methods, and the post-processor is not used during the configuration process, so it is important to explain how the post-processor and MillenniumOS on RRF interact.
 
 ## Post-Processor
 
-The MillenniumOS post-processors are relatively simple - rather than outputting any particularly complex gcode sequences to perform tool changes or probing cycles, they rely on the knowledge that the firmware (RRF, and therefore MillenniumOS macros running on top of it) are best-placed to implement these in a safe manner. The gcode is designed to be simple to read and well commented, so for those of us of a less-trusting nature, it should be easy enough to open in a text editor and understand what it will command your mill to do.
+The MillenniumOS post-processors are relatively simple - rather than outputting any particularly complex gcode sequences to perform tool changes or probing cycles, they rely on the knowledge that the firmware (RRF, and therefore MillenniumOS macros running on top of it) are best placed to implement these safely. The gcode is designed to be simple to read and well commented, so for those with a less-trusting nature, it should be easy enough to open in a text editor and understand what it will command your mill to do.
 
 ![Image showing Post-Process tab of a Fusion360 Setup](../img/mos_usage_step_0.png){: .shadow-dark }
 
 ---
 
-### WCS Offsets / Work Co-ordinate Systems
+### WCS Offsets / Work Coordinate Systems
 
-The **"WCS offset"** setting of the **"Setup"** selects the WCS that will be switched into prior to running the tool paths configured in the Setup.
+The **"WCS offset"** setting of the **"Setup"** selects the WCS used while running the tool paths configured in the Setup.
 
-A value of `0` in this field corresponds to `G54`, or the *first* WCS. We do not allow output of gcode in machine co-ordinates so this field is zero-indexed. A value of 2 corresponds to `G56`, or the *third* WCS.
+A value of `0` or `1` in this field corresponds to `G54`, or the *first* WCS, as we do not allow the output of gcode in machine coordinates (`G53`).
 
-The **"WCS offset"** setting can be used in a number of ways.
+This follows industry standards so that offsets 2 and 3 correspond to `G55` and `G56` respectively.
+
+The **"WCS offset"** setting can be used in several ways.
 
 ![Post-Process tab of a Fusion360 Setup](../img/mos_usage_step_1.png){: .shadow }
 
 ---
 
 #### Single work-piece, multiple operations on different faces
 
-The below flow chart indicates a number of machining operations that are performed on different faces of the same work-piece, e.g. machining the top face of a part, and then flipping it over to machine from the opposite side.
+The below flow chart indicates several machining operations that are performed on different faces of the same workpiece, e.g. machining the top face of a part, and then flipping it over to the machine from the opposite side.
```

Comment:
```suggestion
The below flow chart indicates several machining operations that are performed on different faces of the same workpiece, e.g. machining the top face of a part, and then flipping it over to mill from the opposite side.
```

`to the machine` changes the meaning of the sentence, this is talking about machining the stock in another orientation.